radixdlt-cli/src/main/groovy/com/radixdlt/cli/SendMessage.groovy

```diff
@@ -0,0 +1,35 @@
+package com.radixdlt.cli
+
+import com.radixdlt.client.application.RadixApplicationAPI
+import com.radixdlt.client.atommodel.accounts.RadixAddress
+import org.radix.utils.RadixConstants
+import picocli.CommandLine
+
+@CommandLine.Command(name = "send-message", mixinStandardHelpOptions = true,
+        description = "Send Message")
+class SendMessage implements Runnable {
+
+    @CommandLine.ArgGroup(exclusive = true, multiplicity = "0..1")
+    Composite.IdentityInfo identityInfo
+
+    @CommandLine.Option(names = ["-m", "--message"], paramLabel = "MESSAGE", description = "message to send", required = true)
+    String messageString
+
+    @CommandLine.Option(names = ["-d", "--address"], paramLabel = "ADDRESS", description = "Address to which message is sent", required = true)
```

Comment:
change to `recipient`? _"address"_ is ambiguous, because could mean address of sender..?

Reply:
or just `to` ?